adapters/jwplayer/utils.go

```diff
@@ -76,6 +76,71 @@ func GetAppnexusExt(placementId string) json.RawMessage {
 	return jsonExt
 }
 
+type xandrVideoExt struct {
```

Comment:
same comment as other PR that these structs and functions do no feel like general utilities but tied to specific business logic and should go in a xandr.go file maybe

Reply:
addressed